code/__DEFINES/subsystems.dm

```diff
@@ -165,6 +165,7 @@
 // Subsystem fire priority, from lowest to highest priority
 // If the subsystem isn't listed here it's either DEFAULT or PROCESS (if it's a processing subsystem child)
 
+#define FIRE_PRIORITY_PING 10
```

Comment:
you sure you want it this low in priority?

Reply:
This is how Goonchat had it, and it's perfectly fine. A few skipped soft pings are not the end of the world.